src/cljs/rems/administration/workflow.cljs

```diff
@@ -5,14 +5,16 @@
             [rems.application :refer [enrich-user]]
             [rems.autocomplete :as autocomplete]
             [rems.collapsible :as collapsible]
+            [rems.common-util :refer [vec-dissoc]]
+            [rems.config :refer [dev-environment?]]
             [rems.spinner :as spinner]
             [rems.text :refer [text text-format localize-item]]
-            [rems.common-util :refer [vec-dissoc]]
             [rems.util :refer [dispatch! fetch post!]]))
 
 (defn- reset-form [db]
   (assoc db
-         ::form {:rounds []}
+         ::form {:type :auto-approve
+                 :rounds []}
```

Comment:
why have `:rounds []` in the default but not `:handlers []`? I'd prefer to have neither.